packages/nest/src/generators/application/lib/normalize-options.ts

```diff
@@ -43,5 +43,6 @@ export function toNodeApplicationGeneratorOptions(
     tags: options.tags,
     unitTestRunner: options.unitTestRunner,
     setParserOptionsProject: options.setParserOptionsProject,
+    bundler: 'webpack', // Some features require webpack plugins such as TS transformers
```

Comment:
Nest needs webpack for now since esbuild is missing some features. If we have time in the effort we can address it later.